Assets/MixedRealityToolkit/Utilities/MathUtilities.cs

```diff
@@ -515,5 +515,36 @@ public static Vector3 GetDirection(Quaternion rotation)
             return Mathf.Abs(yaw) < horizontalFovHalf && Mathf.Abs(pitch) < verticalFovHalf;
         }
 
+        /// <summary>
+        /// Returns true if a point lies inside the cone described with given parameters, false otherwise.
+        /// The cone is inscribed to a radius equal to the vertical height of the provided FOV.
```

Comment:
```suggestion
```

Suggest deleting this line -- I don't understand what it says in this context (think it made more sense when the comment described a camera method).